build.py

```diff
@@ -120,33 +106,34 @@ def clean():
 # Docs
 # ====
 
-def _sphinx_cmd(packages, cmd):
+def _sphinx_cmd(cmd, extra_pkgs=[]):
     envdir = _env()
-    run('pip', 'install', *packages)
+    _deps()
```

Comment:
Harumph. If tox [supported multiple requirements files](https://bitbucket.org/hpk42/tox/issues/362) there may have be something here. I'm not going to pursue this further, though.